spec/mocha.opts

```diff
@@ -0,0 +1,2 @@
+--require spec/setup.js
+--recursive
```

Comment:
The first seems more readable to me but I'm on board with moving this to config at least until we have enough opts to warrant an external file. 

```
--require spec/setup.js
--recursive
--grep *.spec.js
```

vs.

```
mocha --require spec/setup.js --recursive spec --grep *.spec.js
```